lib/IRremoteESP8266/IRremoteESP8266.cpp

```diff
@@ -1314,8 +1303,8 @@ void ICACHE_FLASH_ATTR IRsend::sendMitsubishiAC(unsigned char data[]) {
 // IRRecv------------------------------------------------------
 
 extern "C" {
-  #include "user_interface.h"  // NOLINT(build/include)
   #include "gpio.h"
+  #include "user_interface.h"
```

Comment:
Is there a reason why you didn't use #include <user_interface.h> and #include <gpio.h> here?